bramkouwen/Music-Mixer
# Adding Images

## What does the PR do?
* It adds images

## How do you check this PR?
* Pull down the branch
*  See added images

## Screenshots 
None.
